src/v3/src/transformer/layout/idp/transformIdvIdpAuthenticator.ts

```diff
@@ -44,6 +49,37 @@ export const transformIdvIdpAuthenticator: IdxStepTransformer = ({
     },
   };
 
+  const termsOfUseDescription: DescriptionElement = {
+    type: 'Description',
+    contentType: 'subtitle',
+    options: {
+      variant: 'subtitle1',
+      content: loc('oie.idv.idp.description.termsOfUse', 'login', undefined, {
+        $1: {
+          element: 'a',
+          attributes: {
+            class: 'terms-of-use', href: 'https://withpersona.com/legal/terms-of-use', target: '_blank', rel: 'noopener noreferrer',
```

Comment:
Same comment as earlier about changing `.terms-of-use` and `.privacy-policy` to be `data-se` if they're for testing only